metrics2/metrics2.go

```diff
@@ -0,0 +1,103 @@
+package metrics2
```

Comment:
Just out of curiosity, why call this `metrics2` and not just `metrics`? If needs to run in parallel with the old metrics library, people can just import this one as:

```go
import metrics2 "github.com/remind101/pkg/metrics
```